sphinx/users/imagej/installing.rst

```diff
@@ -10,7 +10,7 @@ Installing Bio-Formats in ImageJ
     <http://help.openmicroscopy.org/imagej.html>`_ useful
     for getting you started with both plugins at the same time.
 
-Once you `download <https://imagej.nih.gov/ij/download.html>`__ and
+Once you `download <https://imagej.net/downloads>`__ and
```

Comment:
```suggestion
Once you `download <https://imagej.net/ij/download.html>`__ and
```
`imagej.net/ij/download.html` is the equivalent of the previously-referenced download page. imagej.net/downloads references Fiji as well, so may be confusing since the install instructions here are known to be incompatible with Fiji.